modules/OCPP201/manifest.yaml

```diff
@@ -24,6 +24,10 @@ config:
     description: If true websocket can be disconnected and connected externally
     type: boolean
     default: false
+  MessageQueueResumeDelay:
```

Comment:
Probably makes sense to have this in the OCPP (1.6) module as well

Reply:
We would need to extend the libocpp PR a bit as well, in order to allow passing that setting to the 1.6 chargepoint. Should we do that as part of this PR?

Reply:
Done